en/guide/error-handling.md

```diff
@@ -42,6 +42,49 @@ app.get('/', function (req, res, next) {
 })
 ```
 
+Starting with Express 5, route handlers and middleware that return a Promise will call 
+`next(value)` automatically when they reject or throw an error.  For example:
+
+```js
+app.get('/user/:id', function (req, res, next) {
+  return getUserById(req.params.id).then(function (user) {
+    res.send(user)
+  })
+})
+```
+
+If `getUserById` throws an error or rejects, `next` will be called with either the thrown error or the rejected value.  
+If no rejected value is provided, `next` will be called with the string `Rejected promise`.
```

Comment:
> will be called with the string `Rejected promise`

AFAIK it is called with an actual `Error` object and not with a string.

Reply:
you're 100% correct. I'll update this